driver/test/performance_it.cpp

```diff
@@ -60,24 +55,79 @@ class PerformanceTest
     SQLUINTEGER driver_log = SQL_OPT_TRACE_ON;
 };
 
-#ifdef NDEBUG
+// API call that doesn't reach the driver, i.e., this is effectively (somewhat incomplete) Driver Manager overhead, just for the reference.
+TEST_F(PerformanceTest, ENABLE_FOR_OPTIMIZED_BUILDS_ONLY(UnimplementedAPICallOverhead)) {
+    constexpr std::size_t call_count = 1'000'000;
+    const auto tstr = fromUTF8<SQLTCHAR>("");
+    auto * tstr_wptr = const_cast<SQLTCHAR * >(tstr.c_str());
+
+    // Verify that SQLStatistics() is not implemented. Change to something else when implemented.
+    {
+        const auto rc = SQLStatistics(hstmt, tstr_wptr, 0, tstr_wptr, 0, tstr_wptr, 0, SQL_INDEX_ALL, SQL_ENSURE);
+        if (rc != SQL_ERROR) {
+            throw std::runtime_error(
+                "SQLStatistics return code: " + std::to_string(rc) +
+                ", expected SQL_ERROR (" + std::to_string(SQL_ERROR) +
+                ") - a function that is not implemented by the driver"
+            );
+        }
+
+        // TODO: fix this, the following extra check works reliably only on UnixODBC with matching encoding versions of driver and app.
+/*
+        const auto diag_str = extract_diagnostics(hstmt, SQL_HANDLE_STMT);
+        ASSERT_THAT(diag_str, ::testing::HasSubstr("Driver does not support this function"));
+*/
+    }
+
+    START_MEASURING_TIME();
+
+    for (std::size_t i = 0; i < call_count; ++i) {
+        SQLStatistics(hstmt, tstr_wptr, 0, tstr_wptr, 0, tstr_wptr, 0, SQL_INDEX_ALL, SQL_ENSURE);
+    }
+
+    STOP_MEASURING_TIME_AND_REPORT();
```

Comment:
I suggest also report iterations count, so you can properly compare if someone is to change that number later.

Reply:
Makes sense.

Reply:
Done.